tests/coreclr/p1-net6.0-ubuntu/Makefile

```diff
@@ -90,10 +90,12 @@ run:
 	# $(MAKE) profiler-custom-test-runs
 	# Model number 106 or 108 indicates IceLake machines
 	if cat /proc/cpuinfo | grep -m 1 "model.*:" | grep '106\|108' > /dev/null; then \
-	$(TEST_RUNNER) $(MYST_EXEC) config_1g.json 300 ext2 pr1-icelake-1g-passed 4 100; \
-	$(TEST_RUNNER) $(MYST_EXEC) config_8g.json 2400 ext2 pr1-icelake-8g-passed 1 100; \
+		$(TEST_RUNNER) $(MYST_EXEC) config_1g.json 300 ext2 pr1-icelake-1g-passed 4 100; \
+		$(TEST_RUNNER) $(MYST_EXEC) config_8g.json 2400 ext2 pr1-icelake-8g-passed 1 100; \
+		$(STAT) pr1-only-tests 96; \
+	else \
+		$(STAT) pr1-only-tests 90; \
```

Comment:
Do we know why sometimes we are able to achieve 96% on coffee lake?

Reply:
Going by https://oe-jenkins-dev.westeurope.cloudapp.azure.com/job/Mystikos/job/P1-Tests/ , it doesn't look like we ever were passing on CFL.
cc: @CyanDevs @rs--

Reply:
The builds from Jan 29 - Feb 28 were actually passing, but the failure was silent because the failure wasn't caught when tests scored < 96%.

E.g. From https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/Mystikos%2FStandalone-Pipelines%2FDotNet-P1-Tests-Pipeline/detail/DotNet-P1-Tests-Pipeline/301/pipeline/53
```
[2022-02-13T11:41:28.073Z] Tests passed: 5232
[2022-02-13T11:41:28.073Z] Pass percentage: 91 %
...
[2022-02-13T11:41:28.074Z] make[1]: Leaving directory '/home/oeadmin/workspace/Mystikos/Standalone-Pipelines/DotNet-P1-Tests-Pipeline/solutions/coreclr-p1'
[2022-02-13T11:41:28.074Z] make: Leaving directory '/home/oeadmin/workspace/Mystikos/Standalone-Pipelines/DotNet-P1-Tests-Pipeline/solutions/coreclr-p1'
```

Now there are more tests and still scored < 96% but the failure is caught.


E.g. From https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/Mystikos%2FStandalone-Pipelines%2FDotNet-P1-Tests-Pipeline/detail/DotNet-P1-Tests-Pipeline/511/pipeline
```
[2022-03-28T08:25:42.828Z] Tests passed: 4991/5522
[2022-03-28T08:25:42.828Z] Pass percentage: 90 %
...
[2022-03-28T08:25:42.828Z] Failed to pass at least 96% - Pass percentage: 90% 4991/5522
[2022-03-28T08:25:42.828Z] Makefile:86: recipe for target 'run' failed
[2022-03-28T08:25:42.828Z] make[1]: *** [run] Error 1
[2022-03-28T08:25:42.828Z] make[1]: Leaving directory '/home/oeadmin/workspace/Mystikos/Standalone-Pipelines/DotNet-P1-Tests-Pipeline/tests/coreclr/p1-net6.0-ubuntu'
[2022-03-28T08:25:42.828Z] Makefile:105: recipe for target 'tests' failed
[2022-03-28T08:25:42.828Z] make: *** [tests] Error 2
[2022-03-28T08:25:42.828Z] make: Leaving directory '/home/oeadmin/workspace/Mystikos/Standalone-Pipelines/DotNet-P1-Tests-Pipeline/tests/coreclr/p1-net6.0-ubuntu'
script returned exit code 2
```

Reply:
Thanks for sharing that @CyanDevs, that timeline makes sense. The test lists got updated on Feb 14, and there were a couple of fixes made to the test setup (last on 8th March). The 3 weekend runs after that, all have p1-cfl failing.
Also it looks like we are running .net p1 only on v2 cfl vms, and not both icelake and cfl.
cc: @radhikaj